src/transport/concentration_model.cc

```diff
@@ -123,26 +123,12 @@ ConcentrationTransportModel::ModelEqData::ModelEqData()
 
 
 
-UnitSI ConcentrationTransportModel::balance_units()
```

Comment:
How is this replaced? For the heat model it is moved into constructor. Maybe similar setting is missing for the concentration model.